client/app/services/vms.service.js

```diff
@@ -130,7 +134,8 @@ export function VmsService (CollectionsApi, RBAC) {
       create: RBAC.hasAny(['vm_snapshot_add']),
       delete: RBAC.hasAny(['vm_snapshot_delete']),
       deleteAll: RBAC.hasAny(['vm_snapshot_delete_all']),
-      revert: RBAC.hasAny(['vm_snapshot_revert'])
+      revert: RBAC.hasAny(['vm_snapshot_revert']),
+      instanceRetire: RBAC.hasAny(['instance_control', 'instance_retire'])
```

Comment:
I was (am) guilty of this, making up names for rbac permissions...

It would be best to keep the name as close to the actualproduct feature as possible, so for this line `instance_control: RBAC.hasAny(['instance_control')]`is preferred